src/pages/course.js

```diff
@@ -57,6 +57,14 @@ const Course = props => {
             {props.course.name}
           </h1>
           <ShowForCourseStaff courseId={props.courseId}>
+            <Button
+              color="primary"
+              className="mr-3 mt-3"
+              href={`/api/courses/${props.courseId}/data/questions`}
```

Comment:
This should use `withBaseUrl` from `util/index.ts` to account for the case where the queue is deployed at a path other than the domain root (like in prod, where it's at `queue.illinois.edu/q/`).